src/components/graph/Graph.jsx

```diff
@@ -311,6 +319,12 @@ export default class Graph extends React.Component {
     selector.call(zoomObject).on("dblclick.zoom", null);
   };
 
+  _zoomedEnd = () => {
+    const transform = d3Event.transform;
+
+    this.onDragChange(transform);
```

Comment:
Check if it's defined, the prop might be invalidated/removed on later renders and it's not guaranteed to be present even with the `if` statement introduced above

```suggestion
    this.onDragChange && this.onDragChange(transform);
```